### DISCUSSION TITLE
How to publish to `<username>.github.io` homepage, not `<username>.github.io/<repository>` site?

### DISCUSSION THREAD

**Discussion by lulunac27a** [squidfunk/mkdocs-material#6857] [Q&A]
I want to publish to `<username>.github.io` homepage, not `<username>.github.io/<repository>` site (like `lulunac27a.github.io/` in my GitHub Pages homepage, not `lulunac27a.github.io/<repository>`), but according to this [link](https://squidfunk.github.io/mkdocs-material/publishing-your-site/), I can't find how to publish to homepage of my GitHub Pages site.

**Comment by kamilkrzyskow** [comment]
Hello @lulunac27a,
as per the documentation: 
https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites

You need to create a `lulunac27a.github.io` repository on your account and then GitHub Pages won't add the repository name suffix.
Of course the files for the project will be also hosted inside of that specially named repository.
You can't do it for any repository with another name than the above specifies.

**Reply by kamilkrzyskow** [reply]
Also on the link you've provided: https://squidfunk.github.io/mkdocs-material/publishing-your-site/
there is the link to GitHub Pages: https://pages.github.com/
where it's explained how to setup the user site 👌
